From 9277f15d4126df98d0870b274f7a2e5d0fa82dba Mon Sep 17 00:00:00 2001
From: Stanislaw Kardach <skardach@marvell.com>
Date: Fri, 21 Jun 2019 14:23:13 +0200
Subject: [PATCH 253/767] drivers: spi: octeontx2: add entry of PCI device for
 LOKI

commit fc164f03884b41f1420dc132a7de5b3f38a0123b from
git@git.assembla.com:cavium/WindRiver.linux.git

Enable octeontx2 SPI controller driver on LOKI SoC.

Change-Id: I3e799f670d34cfd7212b371bf4b90c233793e902
Signed-off-by: Stanislaw Kardach <skardach@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/11421
Reviewed-by: Dovrat Zifroni <dovrat@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Tested-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/spi/spi-octeontx2.c | 5 +++++
 drivers/spi/spi-octeontx2.h | 1 +
 2 files changed, 6 insertions(+)

diff --git a/drivers/spi/spi-octeontx2.c b/drivers/spi/spi-octeontx2.c
index f9ffffbec2eb..7ffb9f083c92 100644
--- a/drivers/spi/spi-octeontx2.c
+++ b/drivers/spi/spi-octeontx2.c
@@ -331,6 +331,11 @@ static const struct pci_device_id octeontx2_spi_pci_id_table[] = {
 			 PCI_DEVID_OCTEONTX2_SPI,
 			 PCI_VENDOR_ID_CAVIUM,
 			 PCI_SUBSYS_DEVID_OTX2_95XX) },
+	{ PCI_DEVICE_SUB(PCI_VENDOR_ID_CAVIUM,
+			 PCI_DEVID_OCTEONTX2_SPI,
+			 PCI_VENDOR_ID_CAVIUM,
+			 PCI_SUBSYS_DEVID_OTX2_LOKI) },
+
 	{ 0, }
 };
 
diff --git a/drivers/spi/spi-octeontx2.h b/drivers/spi/spi-octeontx2.h
index d7038522b555..84dfae89a8f8 100644
--- a/drivers/spi/spi-octeontx2.h
+++ b/drivers/spi/spi-octeontx2.h
@@ -7,6 +7,7 @@
 #define PCI_DEVID_OCTEONTX2_SPI 0xA00B
 #define PCI_SUBSYS_DEVID_OTX2_96XX 0xB200
 #define PCI_SUBSYS_DEVID_OTX2_95XX 0xB300
+#define PCI_SUBSYS_DEVID_OTX2_LOKI 0xB400
 
 #define OCTEONTX2_SPI_MAX_BYTES 1024
 #define OCTEONTX2_SPI_MAX_CLOCK_HZ 25000000
-- 
2.31.1

